ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -1,15 +1,69 @@
 # global
-from hypothesis import strategies as st, assume
+from hypothesis import given, strategies as st, assume
 import numpy as np
-
-
 # local
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
     statistical_dtype_values,
 )
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
```

Comment:
E   ImportError: cannot import name 'handle_cmd_line_args' from 'ivy_tests.test_ivy.helpers' (/ivy/ivy_tests/test_ivy/helpers/__init__.py)
you should remove this line